### PR TITLE
CANVAS-49 - Handle crosslists

### PR DESCRIPTION
**Pull request by maxpluez**

### CODE REVIEW

**Comment by rlorenzo** [comment]
@maxpluez I checked out your branch and I cannot save anything in the "Bruincast crosslists". I click "Submit" and nothing happens. 

Then after a while, I get an error and in the console I see this error:

(node:58745) UnhandledPromiseRejectionWarning: BulkWriteError: Cannot create namespace lti-media-resources.crosslists in multi-document transaction.

Similar to the error in https://ucla-ccle.atlassian.net/browse/CANVAS-78

**Comment by rlorenzo** [comment]
@maxpluez Okay, I had to create a collection manually called crosslists, but that should be created if it doesn't exist.

When I save a crosslist I get this ugly popup:

Screen Shot 2020-08-03 at 7 12 33 PM

Can you use the success or error alerts instead? See https://instructure.design/#Alert

**Comment by rlorenzo** [comment]
@maxpluez The tab for Bruincasts still shows 0 even after I make a crosslist. The tab should count all videos available to the course:

Screen Shot 2020-08-03 at 7 14 40 PM

File: src/server/models/mediaquery.js [hunk 1]
```diff
@@ -41,6 +41,64 @@ module.exports.setNotice = async notice => {
...
+    .db(DB_DATABASE)
+    .listCollections()
+    .toArray();
+  if (!collectionList.map(col => col.name).includes('crosslists')) {
+    await client.db(DB_DATABASE).createCollection('crosslists');
+    await client
+      .db(DB_DATABASE)
+      .collection('crosslists')
+      .insertOne({
+        list: ['201-COMSCI32-1', '131A-ART133-1'],
```
Comment (rlorenzo) [inline src/server/models/mediaquery.js:56]: @maxpluez Why is this being hardcoded?

**Comment by rlorenzo** [comment]
@maxpluez This is great, but I don't see any tests written. Please add tests for the crosslisting feature.

**Comment by maxpluez** [comment]
> @maxpluez Do you need to package installed globally? Isn't it installed by yarn?

Yeah at least on my system. migrate-mongo is a cli tool..

**Comment by rlorenzo** [comment]
@maxpluez Works on my local dev instance and added a change to clean things up: https://github.com/ucla/lti-media-resources/pull/7/commits/0eea9e582ca9a138f3b0a99863e1bc9cf1f8479f

**Pull request merged**